src/picknik_dual_ur_arm_config/objectives/request_teleoperation.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="Request Teleoperation">
+  <!-- The integer value used here for teleoperation mode comes from the moveit_studio_sdk_msgs/TeleoperationMode ROS message. -->
+  <BehaviorTree ID="Request Teleoperation" _description="Handles the different variations of teleoperation from the web UI, with the option of interactive user prompts and choosing the initial mode. Should be used as a subtree with port remapping as part of an another Objective." _favorite="false" _hardcoded="false">
+    <Control ID="Sequence">
+      <Action ID="Script" code="teleop_mode := 0"/>
+      <Control ID="Parallel" success_count="1" failure_count="1">
+        <Action ID="DoTeleoperateAction" enable_user_interaction="{enable_user_interaction}" user_interaction_prompt="{user_interaction_prompt}" initial_teleop_mode="{initial_teleop_mode}" current_teleop_mode="{teleop_mode}"/>
+        <Decorator ID="KeepRunningUntilFailure">
+          <Control ID="Sequence">
+            <!-- Closing and opening the gripper -->
+            <Decorator ID="ForceSuccess" _skipIf="teleop_mode != 7">
+              <SubTree ID="Close Right Gripper"/>
+            </Decorator>
+            <Decorator ID="ForceSuccess" _skipIf="teleop_mode != 6">
+              <SubTree ID="Open Right Gripper"/>
+            </Decorator>
```

Comment:
Sadly this is something else (low priority) we need to figure out in this multi-arm initiative. When you hit the "Open/Close Gripper" button, how do we configure which gripper it is?

Reply:
:+1: I've opened a  new issue to track this https://github.com/PickNikRobotics/moveit_studio/issues/6781